pkg/http/api.go

```diff
@@ -943,14 +944,17 @@ func (a *api) onPublish(reqCtx *fasthttp.RequestCtx) {
 		return
 	}
 
+	pubsubName := reqCtx.UserValue(pubsubnameparam).(string)
 	topic := reqCtx.UserValue(topicParam).(string)
 	body := reqCtx.PostBody()
 
+	log.Debugf("Publishing message to pubsub='%s', topic='%s'\n", pubsubName, topic)
```

Comment:
Let's remove this. If someone is logging in Debug in production, it means they're most likely looking for an error and this is a per-operation info level log which will might cause a log-flood, depending on the scale of events coming in.